docs/library/sys.rst

```diff
@@ -90,6 +92,11 @@ Constants
    access this field using ``sys.implementation._build.split("-")``.  The
    elements that are currently used are:
 
+   The *_thread* entry was added in version 1.26.0 and if it exists then the
+   target has the ``_thread`` module.  The target enables the GIL (global
+   interpreter lock) if this attribute equals ``"GIL"``, otherwise the attribute
+   is ``"no-GIL"`` and the target does not enable the GIL.
```

Comment:
Do you think MicroPython will ever have a variant that is non-GIL but thread safe (ala CPython `--disable-gil`?)

If yes then maybe we could make the attribute values `GIL` and `unsafe` (or similar), keeping the door open for a `safe` value in some future release.

Reply:
(Would mean tests which test for no-GIL will need to test `_thread != "GIL", I guess

Reply:
> Do you think MicroPython will ever have a variant that is non-GIL but thread safe

Yes, that's a possibility.  But, I would suspect that we gradually evolve the current no-GIL mode to a more safer behaviour, rather than have separate options for GIL/unsafe-no-GIL/safe-no-GIL.

That said, I'm happy to change "no-GIL" to "unsafe".  Although that could scare users, but OTOH it might push them to read the docs to understand what "unsafe" actually means.

Is there another word that could better describe the situation, eg "racey" or "raceful"?

Reply:
OK, I've now renamed "no-GIL" to "unsafe", and updated the docs.

Also tested again on esp32, rp2 and unix.